packages/rolldown/src/types/rolldown-output.ts

```diff
@@ -20,6 +20,8 @@ export interface SourceMap {
   sources: string[]
   sourcesContent: string[]
   version: number
+  debugId?: string
+  x_google_ignoreList?: number[]
```

Comment:
Rollup does not have this field here, but I think it makes sense to have it here as well.